extensions/teamgantt/src/search.tsx

```diff
@@ -0,0 +1,105 @@
+import { List, getPreferenceValues, Icon, ActionPanel, Action, useNavigation, showToast, Toast } from '@raycast/api';
+import { useEffect, useState } from 'react';
+import { MyPreferences, Task } from './types';
+import { authenticate, findTasks, punchIn } from './api';
+import { groupTasksByProject } from './utils';
+import TaskDetails from './task-details';
+
+export default function Command() {
+  const { push } = useNavigation();
+  const [isLoading, setIsLoading] = useState(true);
+  const [showCompleted, setShowCompleted] = useState(false);
+  const [idToken, setIdToken] = useState<string>();
+  const [query, setQuery] = useState<string>();
+  const [tasks, setTasks] = useState<Task[]>([]);
+  const preferences = getPreferenceValues<MyPreferences>();
+
+  // Auth
+  useEffect(() => {
+    setIsLoading(true);
+
+    authenticate(preferences)
+      .then(({ id_token }) => {
+        setIdToken(id_token);
+      })
+      .catch((e) => {
+        console.log('Failed authenticating', e);
+        showToast({ title: 'Failed to authenticate', message: e.message ?? e.toString(), style: Toast.Style.Failure });
+      })
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  // Find tasks
+  useEffect(() => {
+    if (idToken && query) {
+      setIsLoading(true);
+
+      findTasks({ q: query ? query : undefined, showCompleted }, idToken)
+        .then((tasks) => {
+          setTasks(tasks as Task[]);
+        })
+        .finally(() => setIsLoading(false));
+    }
+  }, [idToken, query, showCompleted]);
+
+  const onSearch = (value: string) => {
+    setQuery(value ? value : undefined);
+  };
+  const grouped = groupTasksByProject(tasks);
+  const groups = Object.keys(grouped).sort((a, b) => (a > b ? 1 : -1));
+
+  return (
+    <List
+      navigationTitle="Tasks"
```

Comment:
Don't change the navigationTitle in the root command - it will be automatically set to the command name. Use navigationTitle only in nested screens to provide additional context.